src/main/java/org/jitsi/jicofo/conference/ParticipantInviteRunnable.java

```diff
@@ -468,8 +468,16 @@ private boolean doInviteOrReinvite(Jid address, Offer offer, ColibriAllocation c
 
     private @NonNull Offer updateOffer(Offer offer, ColibriAllocation colibriAllocation)
     {
-        // Take all sources from participants in the conference.
-        ConferenceSourceMap conferenceSources = meetConference.getSources().copy();
+        ConferenceSourceMap conferenceSources;
+
+        if (!participant.hasSsrcRewritingSupport()) {
+            // Take all sources from participants in the conference.
+            conferenceSources = meetConference.getSources().copy();
+        } else {
+            // Bridge will signal sources in this case.
+            conferenceSources = new ValidatingConferenceSourceMap();
```

Comment:
You can use a regular `ConferenceSourceMap` here, no validation (checking for ssrc conflicts, etc) needed.